### DISCUSSION TITLE
Active checkers

### DISCUSSION THREAD

**Discussion by Midya-ELTE** [find-sec-bugs/find-sec-bugs#695] [Q&A]
Hi all, how to know if the checker of Find Security Bug is active or not by default?

**Comment by JuditKnoll** [comment]
Hi!
In the [findbugs.xml](https://github.com/find-sec-bugs/find-sec-bugs/blob/master/findsecbugs-plugin/src/main/resources/metadata/findbugs.xml) file at the Detector definition if `disabled="true"` is given, then it's disabled by default. The default value of the `disabled` parameter is false, so if it's not given, the detector is active.
As far as I can see, currently none of the detectors are disabled, all are active.
